src/main/java/org/jenkinsci/plugins/compressbuildlog/BuildLogCompressor.java

```diff
@@ -28,7 +34,7 @@ public BuildLogCompressor() {
 
         @Override
         public String getDisplayName() {
-            return "Compress Build Log";
+            return "Compress Build Logs";
```

Comment:
Yeah, it was the groovy template that I later realized I needed to change -- just thought it looked better on the /configure page. :smile: